cfgov/unprocessed/js/routes/on-demand/simple-chart/simple-chart.js

```diff
@@ -117,6 +118,40 @@ function makeChartOptions( data, dataAttributes ) {
     };
   }
 
+  if (projectedMonths) {
+  // Convert the number of projected months into a timestamp
+  const lastChartDate = defaultObj.series[0].data.at(-1).x;
```

Comment:
The indentation of this `if` statement is a little off. The innards need to be indented one level.